frontend/public/components/dashboard/events-card/events-body.tsx

```diff
@@ -9,10 +9,10 @@ import { EventKind } from '../../../module/k8s';
 
 export const EventsBody: React.FC<EventsBodyProps> = ({ events, filter }) => {
   let eventsBody;
-  if (!(events && events.loaded)) {
-    eventsBody = <div className="co-events-card__body-loading"><LoadingInline /></div>;
-  } else if (events.loadError) {
+  if (_.get(events , 'loadError')) {
```

Comment:
nit: I have a slight preference for

```suggestion
  if (events && events.loadError) {
```

to match the `if` just below and for better type checking

Reply:
fixed